shtp.c

```diff
@@ -63,8 +64,13 @@ typedef struct shtp_App_s {
     char appName[SHTP_APP_NAME_LEN];
 } shtp_App_t;
 
-typedef struct shtp_AppListener_s {
+typedef struct shtp_AppListener_s
+{
+#ifdef CONFIG_SHTP_CONST_NAMES
+    const char *appName;
```

Comment:
Can we submit this change upstream?

Reply:
Haven't had any luck getting them approved. But sure.